core/test/unit/web/middleware/image/normalize_spec.js

```diff
@@ -68,4 +69,15 @@ describe('normalize', function () {
             done();
         });
     });
+
+    ['.gif', '.svg', '.svgz'].forEach(function (extension) {
```

Comment:
@kirrg001 haven't seen such a syntax (iterative `it` creation) elsewhere in the codebase. Is there some other preferred way for such check or it's fine to use this way? :thinking:

Reply:
I think this is fine. We don't have any other example in Ghost.
Bookshelf uses the same style to test different database types, [see](https://github.com/bookshelf/bookshelf/blob/master/test/integration.js#L52).